examples/official-site/sqlpage/migrations/32_tracking_update.sql

```diff
@@ -0,0 +1,17 @@
+INSERT INTO parameter (
+        component,
+        name,
+        description,
+        type,
+        top_level,
+        optional
+    )
+VALUES (
+        'tracking',
+        'center',
+        'Whether to center the component.',
+        'BOOLEAN',
+        TRUE,
+        TRUE
+    );
+
```

Comment:
Can we keep that in the existing tracking component documentation file ?

The database behind sql.ophir.dev is in-memory anyway and migrations are reapplied on every deploy.

Reply:
ok, I will merge the two files :)